README.md

```diff
@@ -27,23 +74,144 @@ mtx.myTranspose();
         [2, 4, 6]
        ]
 ```
+const myTranspose = function(mtx){
+  let result = []
+  for(i=0; i< mtx.length; i+2){
+    result.push(mtx[i])
+  }
+}
 
 7. Use `myMap` to build a new array where every element is incremented by 10.
-8. Use `myMap` to build a new array where every non-string element is converted to an empty string. 
+
+let arr = [1,2,3];
+arr.map(el => {
+  return el + 10;
+  });
+
+8. Use `myMap` to build a new array where every non-string element is converted to an empty string.
+
+let arr = ['a', 2, 5, 'b']
+
+arr.map(el=> {
+  let result = []
+  if (arr[i] != number){
+  return result.push(arr[i])
+  } else {
+    return result.push("")
+{
+    })
+
 9. Use `myFilter` to get only the even elements in an array.
-10. Use `myEvery` to check if all elements in the array are the same. 
-11. Use `myReduce` to return the sum of every element in an array. 
-12. Use `myMap` to build a new array that doubles each element. Then chain `myReduce` to find the product of all the elements. 
-13. Use `myReduce` to find the largest number in an array. 
-14. Write a function `elementDivisibleBy` with parameters `divisor` and `arr`.
+
+Array.prototype.myFilter = function(arr) {
+  let output = [];
+  if(arr[i]%2 === 0){
+    output.push(arr[i])
+  }
+  return output
+}
+
+10. Use `myEvery` to check if all elements in the array are the same.
+
+Array.prototype.myEvery = function(arr){
+  for(i=0;i<arr.length;i++){
+    for(j=0;j<arr.length;j++){
+      if(arr[i]===arr[j]){
+        return true
+      }
+    }
+  }
+}
+
+
+11. Use `myReduce` to return the sum of every element in an array.
+
+Array.prototype.myReduce = function(arr){
+  result = 0
+  for(i=0;i<arr.length;i++){
+    result += arr[i]
+  }
+}
+
+12. Use `myMap` to build a new array that doubles each element. Then chain `myReduce` to find the product of all the elements.
+
+Array.prototype.myMap.myReduce = function(arr){
```

Comment:
So, these latter-half problems aren't about defining stuff on the prototype anymore -- they're about implementing the methods that you defined in the first half